engine/src/main/java/org/hibernate/search/engine/metadata/impl/BridgeDefinedField.java

```diff
@@ -19,12 +22,24 @@
 	private final BackReference<DocumentFieldMetadata> sourceField;
 
 	/*
-	 * Here we cannot use DocumentFieldPath because metadata-providing field bridges
-	 * only provide the absolute name, without telling the prefix from the relative name.
+	 * Here we cannot use DocumentFieldPath because metadata-providing field bridges only provide the absolute name,
+	 * without telling the prefix from the relative name.
 	 */
 	private final String absoluteName;
 	private final FieldType type;
 
+	/*
+	 * Backends have specific field properties (Elastic search dynamic mapping for example). this map will group the
+	 * properties per backend.
```

Comment:
I'm not sure what you mean here. Specifically what is  "backend" meaning in this context?